lib/view_component/previewable.rb

```diff
@@ -7,6 +7,12 @@ module Previewable
     extend ActiveSupport::Concern
 
     included do
+      # Set a custom default preview layout through app configuration:
```

Comment:
Can we add details that we're using this preview layout for the preview index pages too?